build.gradle

```diff
@@ -1,5 +1,6 @@
 plugins {
     id 'java'
+    id("io.swagger.core.v3.swagger-gradle-plugin") version "2.2.21"
```

Comment:
If you want to try this, you might want to try to add `implementation 'org.springdoc:springdoc-openapi-starter-webmvc-ui:2.5.0'` to dependencies